backend/wmg/data/wmg_cube.py

```diff
@@ -191,16 +193,16 @@ def reduce_X(tdb_group, start_time, cube_indices, *accum):
             "py.init_buffer_bytes": 512 * MB,
             "py.exact_init_buffer_bytes": "true",
         }
-        with tiledb.open(f"{tdb_group}/raw", ctx=create_ctx(config_overrides=cfg)) as X:
-            iterable = X.query(return_incomplete=True, order="U", attrs=["data"])
+        with tiledb.open(f"{tdb_group}/{INTEGRATED_ARRAY_NAME}", ctx=create_ctx(config_overrides=cfg)) as X:
+            iterable = X.query(return_incomplete=True, order="U", attrs=["rankit"])
             future = None
             for i, result in enumerate(iterable.df[:]):
-                logger.info(f"reduce raw X, iter {i}, {time.time() - start_time}")
+                logger.info(f"reduce integrated expression data, iter {i}, {time.time() - start_time}")
                 if future is not None:
                     future.result()  # forces a wait
                 future = tp.submit(
                     coo_cube_pass1_into,
-                    result["data"].values,
```

Comment:
one of the key fixes (note that we don't even store "data" anymore, so no possibility of mistakenly using it now)